Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabDomUtils.cpp

```diff
@@ -544,6 +544,47 @@ namespace AzToolsFramework
 
                 return prefabBuffer.GetString();
             }
+
+            bool AddOrUpdateNestedInstance(
+                PrefabDom& prefabDom,
+                const InstanceAlias& nestedInstanceAlias,
+                PrefabDomValueConstReference nestedInstanceDom)
+            {
+                auto instancesMember = prefabDom.FindMember(PrefabDomUtils::InstancesName);
+                if (instancesMember == prefabDom.MemberEnd())
```

Comment:
Is the fact that this code is called without there being a member that lists the instances an indicator of a bug somewhere else ?

Reply:
Not a bug. The "Instances" array is only added when the first nested instance wants to be added.